.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 # Defining default values for all jobs
 defaults: &defaults
     docker:
-        - image: circleci/node:8.9.4-browsers
+        - image: circleci/node:9.8.0-browsers
```

Comment:
Why this version? From what I can tell LTS went from `8.15.1` to `10.13.0`. Current LTS is `10.15.3`. We should use one of those versions.

Reply:
Yup. This was the “quick” solution. Just got word back from the CLI team what node version they support. Will also update the node version in package.json for alignment.

Reply:
Updated to `circleci/node:lts-browsers` - that way we get automated maintenance once the LTS version moves forward. Also updated package.json to reflect new versions.